src/components/RedirectionNotificationModal.js

```diff
@@ -1,72 +1,160 @@
 import Modal from "@mui/material/Modal";
+import { Link } from "react-router-dom/cjs/react-router-dom";
 import CloseRoundedIcon from "@mui/icons-material/CloseRounded";
 import * as s from "../components/RedirectionNotificationModal.sc";
+import { isMobile } from "../utils/misc/browserDetection";
 
 //This modal is used in the ArticlePreview component
 
 //TODO: Further refactor, e.g after testing and making sure users
 //understand text and wording, turn strings into variables and move to definitions.js
 
 export default function RedirectionNotificationModal({
+  api,
   article,
   open,
-  handleClose, //handleClose function defined in the ArticlePreview.js, passed as a prop
+  handleClose,
+  selectedDoNotShowRedirectionModal, //related to the "Do not show" checkbox selection
+  setSelectedDoNotShowRedirectionModal, //related to the "Do not show" checkbox selection
+  setOpenedExternallyWithoutModal, //related to the modal use based on the "Do not show" selection
```

Comment:
Is this really needed? 
Isn't it that the decision of whether something should be opened externally w/o modal should be done at the individual article level? In that case, it can't be a state in the FindArticles component which refers to all the articles. 
 
The comment suggest sthat it's derived from `doNotShowRedirectio_checkbox`. In this case, we should just use that.